main.js

```diff
@@ -0,0 +1,141 @@
+/* eslint no-param-reassign: ["error", { "props": false }] */
+const boxes = document.querySelectorAll('.box');
+
+const Player = (name, symbol) => ({ name, symbol });
+const player1 = Player('Player 1', 'X');
+const player2 = Player('Player 2', 'O');
+
+const gameBoard = (() => {
+  let board = new Array(9).fill(0);
+
+  const disable = () => {
+    boxes.forEach((box) => {
+      box.disabled = true;
+    });
+  };
+
+  const restart = () => {
+    boxes.forEach((box) => {
+      box.disabled = false;
+      box.value = '';
+    });
+    board = new Array(9).fill(0);
+  };
+
+  const turn = () => {
+    const zero = board.filter(a => a === 0);
+    if (zero.length % 2 === 0) {
+      return player2;
+    }
+    return player1;
+  };
+
+  const isFull = () => {
+    const zero2 = board.filter(a => a === 0);
+    if (zero2.length === 0) {
+      return true;
+    }
+    return false;
+  };
+
+  const checkWinner = () => {
+    const combos = [
```

Comment:
These combos is better suited as a main data attribute for our `board` rather than just here in the `checkwinner` method.